in_text/text_04_predictions.yml

```diff
@@ -236,75 +279,90 @@ entities:
       attr-def: >-
         Date that the forecast was issued.
       attr-defs: This data release
-      data-min: XX # will complete when we have latest vbersion of this dataset
-      data-max: XX # will complete when we have latest version of this dataset
-      data-units: NA
+      data-min: 2019-01-03 
+      data-max: 2022-09-22 
+      data-units: date
     -
       attr-label: time
       attr-def: >-
         Valid date of forecast
       attr-defs: This data release
-      data-min: XX # will complete when we have latest vbersion of this dataset
-      data-max: XX # will complete when we have latest vbersion of this dataset
-      data-units: NA
+      data-min: 2019-01-03 
+      data-max: 2022-09-29 
```

Comment:
yes because for the last issue_time we are making a forecast 7 days into the future so there will be 7 more dates in the `time` range than the `issue_time` range